drivers/bridge/bridge.go

```diff
@@ -614,6 +660,12 @@ func (d *driver) DeleteNetwork(nid types.UUID) error {
 		return err
 	}
 
+	// Remove inter-network communication rules
+	err = isolateNetworks(d, n.bridge.bridgeIPv4, getV6Network(n.config, n.bridge), false)
+	if err != nil {
+		return err
+	}
+
 	// Programming
 	err = netlink.LinkDel(n.bridge.Link)
```

Comment:
I think we should reinstall the INC rules in the defer method above if the call to `netlink.LinkDel` fails.